nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2189,7 +2189,7 @@ private boolean mayBeNullExpr(VisitorState state, ExpressionTree expr) {
       return false;
     }
     // the logic here is to avoid doing dataflow analysis whenever possible
-    Symbol exprSymbol = ASTHelpers.getSymbol(expr);
+    @Nullable Symbol exprSymbol = ASTHelpers.getSymbol(expr);
```

Comment:
Nullability annotations should not be needed in local variables. If anything, this means we might indeed need a model for `ASTHelpers.getSymbol(...)` (or, presumably, we just wait for EP to mark it `@Nullable`, given the discussion in #733 )